Quadriga/src/main/java/edu/asu/spring/quadriga/service/workbench/impl/ProjectCollaboratorManager.java

```diff
@@ -92,4 +97,25 @@ public ProjectCollaboratorDTOPK createNewCollaboratorDTOPK(String id, String col
     public ICollaboratorDAO<ProjectCollaboratorDTO> getCollaboratorDao() {
         return projectCollabDAO;
     }
-}
+
+    @Override
+    public List<String> getProjectOwnerAndAdminNames(IProject project) {
```

Comment:
remove this method and use getCollaborators+getOwner instead in your controller. This method a) just blows up the manager, when all you need is already there. And b) just returning a list of usernames makes this method very unreusable. If someone else needs a list but also wants to know what roles the users have, then this method can't be reused. Also, you probably want to know in the webpage if a user is the owner or just and admin. This method doesn't tell you that.